paddle/fluid/inference/tests/api/analyzer_int8_image_classification_tester.cc

```diff
@@ -28,6 +28,7 @@ void SetConfig(AnalysisConfig *cfg) {
   cfg->SwitchSpecifyInputNames();
   cfg->SetCpuMathLibraryNumThreads(FLAGS_paddle_num_threads);
   cfg->EnableMKLDNN();
+  cfg->pass_builder()->AppendPass("fc_mkldnn_pass");
```

Comment:
How about removing this line at first to pass the ci. And fix int8 UT in the next PR?

Reply:
Sure, I'll do it.